core/frontend/src/extension/providers/ExtensionLoadScript.ts

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/**
+ * Executes an extension's bundled javascript module.
+ * First attempts an ES6 dynamic import,
+ * second attempts a dynamic import via a script element as a fallback.
+ * Used by remote and service Extensions.
+ * Throws an error if the module does not have a default or main function to execute.
+ * @internal
+ */
+export async function loadScript(jsUrl: string): Promise<any> {
+  // Warning "Critical dependency: the request of a dependency is an expression"
+  // until that is resolved, leave code commented:
+  // const module = await import(/* webpackIgnore: true */jsUrl);
+  // return execute(module);
```

Comment:
@kckst8 @calebmshafer be aware that I had to comment out the original dynamic import. Otherwise, the build will not pass (specifically, the extract-api step) because of a warning "Critical Dependency: the request of a dependency is an expression". I'm still trying to figure out a way around this... I don't think we should just ignore the warning, do we?

Reply:
> @kckst8 @calebmshafer be aware that I had to comment out the original dynamic import. Otherwise, the build will not pass (specifically, the extract-api step) because of a warning "Critical Dependency: the request of a dependency is an expression". I'm still trying to figure out a way around this... I don't think we should just ignore the warning, do we?

It's likely because we are importing from a variable. I think it's safe to disable if possible. Is it coming from tsc or the linter?

Reply:
Actually I think its coming from Webpack? Is that possible? I tried ts-ignore and lint ignore comments but both did not work.

Reply:
is this something that will prevent runtime extensions from working in general? If not, is there any future breaking change that would come from re-introducing this?

I'm fairly certain the answer is no for both. If that's the case, let's leave it out for now and merge this PR for the 3.2.0 release and file this to follow up on in 3.3.0.

Reply:
Yup, I agree. Lets leave it out for now, and it shouldn't break anything once I figure out a way to add it back without the warning.

Reply:
cool, going to resolve this and finish my review.

Reply:
Can you make some issues for the various things talked about in this thread?

Reply:
Yup, I'm on it!